views/add-recipe.handlebars

```diff
@@ -0,0 +1,37 @@
+<section class="container">
```

Comment:
Code added by Heather before I branched off of her branch, but I added styling here.